### DISCUSSION TITLE
Calibration datasets for the model

### DISCUSSION THREAD

**Discussion by PennyWieser** [ComputationalThermodynamics/MAGEMin#107] [General]
Hello, first, thanks for writing such an awesome tool!

I am trying to compare MELTS (all versions) and Holland models (through MAGEMin) to see which model does a better job of predicting Opx stability in mafic systems. I have compiled Opx-bearing experiments for my system of interest (hawaii), and have done the equivalent of what MELTS calls an equilibrate calculation (e.g. specify P, T, fo2, H2O, bulk composition), and see what model stabilizes Opx in the most experiments at the specified conditions. MageMIN is the clear winner. However, I wanted to check if the experiments I got from LEPR were used in anyway when recalibrating the Holland model, e.g. if someone already tested these, said 'huh, it doesnt stabilize Opx, lets tweak something'. Is there a list of papers that were used in this way? 

Thanks so much! 

Penny Wieser

**Comment by boriskaus** [comment]
Hi Penny,

thanks for your encouragement! We wanted to implement the MELTS database as well in MAGEMin, but found the source code to be rather cryptic which is why we have not made much progress on that end. If you, or someone else, wants to help with this, it would be fantastic so we can include it all in one place.
As to your actual question, I think Nico is the better person to answer that..

**Comment by NicolasRiel** [comment]
Hi Penny,

In terms of experimental dataset used by Holland et al. I don't have much information.

If you use the Igneous database I would suspect that some info are available here https://academic.oup.com/petrology/article/59/5/881/4996852 as the newest opx model was developed for this database.

If you use the Metabasite database you would have to refer to the Green et al., 2016 paper. https://onlinelibrary.wiley.com/doi/10.1111/jmg.12211

If you don't find the information I suggest you contact the THERMOCALC list through the mailing list https://groups.io/g/hpxeosandthermocalc I am sure the people of the mailing list knows more about this!

Hope this helps,

Don't hesitate to tell us if you find that something is missing or must be improved in MAGEMin!
